pages/403.html

```diff
@@ -0,0 +1,13 @@
+<!--
+  .. title: 403 Forbidden
+  .. slug: 403
+  .. errdesc: Page Forbidden
+  .. date: 1970-01-01
+  .. description: You're not allowed to access that page.
+  .. pretty_url: False
+  .. template: error.tmpl
+-->
+
+<p>Please check the URL. If you clicked a link and ended up here, please
```

Comment:
I would say "...a link **on cantera.org** and ended up here..." to avoid encouraging people to tell us about dead links elsewhere on the internet.

Reply:
Good point!